cryptoconditions/types/base_sha256.py

```diff
@@ -1,26 +1,19 @@
-from abc import abstractmethod
+import hashlib
 
 from cryptoconditions.fulfillment import Fulfillment
-from cryptoconditions.lib.hasher import Hasher
 
 
-class BaseSha256Fulfillment(Fulfillment):
+class BaseSha256(Fulfillment):
```

Comment:
Why do we need this class? It seems that this method could just be added to the `Fulfillment` class